service/gcs/bridge/bridge_unit_test.go

```diff
@@ -0,0 +1,619 @@
+package bridge
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"io"
+	"strings"
+	"sync"
+	"testing"
```

Comment:
Are you planning on using `testing` in the long term for these tests, or switching to ginkgo?